source/blog/2018-12-07-the-ember-times-issue-76.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: The Ember Times - Issue No. 76
-author: Chris Ng, Jessica Jordan, Amy Lam, the crowd
+author: Chris Ng, Jessica Jordan, Amy Lam, Alon Bukai the crowd
```

Comment:
there is a comma missing, no?

Reply:
@spruce we removed `the crowd` in the post, should be ok in the main file [here](https://github.com/emberjs/website/blob/master/source/blog/2018-12-07-the-ember-times-issue-76.md)? note: we don't always follow oxford comma